stubs/django/template/base.pyi

```diff
@@ -0,0 +1,2 @@
+class Template:
+    pass
```

Comment:
Should `Template.__init__()` be here too, with a corresponding model?
https://github.com/django/django/blob/1cba3207867fb168788ad6120770d0398b6b2807/django/template/base.py#L144

Reply:
Makes sense. Added. Please check the updated comment above.